### DISCUSSION TITLE
What is academic programming track?

### DISCUSSION THREAD

**Discussion by rokokol** [codewars/codewars.com#2953] [Q&A] [answered]
Hello! Could someone explain me what does academic mean? I totally new in IT and I could not find any info about it 🙏
![изображение](https://user-images.githubusercontent.com/121713022/221413836-dad21ba3-dc97-41ee-ba2a-069bb2721ece.png)

**Comment by hobovsky** [comment, marked as answer]
This would generally mean that you are interested with programming in a (loosely defined) context of teaching, education,  academic research, etc.

**Reply by rokokol** [reply]
Thank you!
